docs/StyleGuide.md

```diff
@@ -419,14 +421,6 @@ the rules below, it is good practice to update the code's style to match them.
             } = args
       ```
 
-      If the field names of the `Args` type are named appropriately, using
-      `RecordWildCards` is also acceptable, see also (17).
-
-      ```haskell
-      foo Args {..} =
-          ..
-      ```
-
```

Comment:
Maybe we should mention somewhere that we prefer not to use `RecordWildCards`?

Reply:
I agree. Maybe [here](https://github.com/input-output-hk/ouroboros-consensus/pull/24/files#r1166667492).

Reply:
Done